python/.pylintrc

```diff
@@ -0,0 +1,561 @@
+[MASTER]
```

Comment:
Where do these pylint rules come from?

Reply:
That's generated by default using `pyling` tool and without any change now. We can update that later as we required such as ignore some files.